src/ontology/ro-edit.owl

```diff
@@ -6147,6 +6149,22 @@ AnnotationAssertion(rdfs:label obo:RO_0016002 "has disease")
 AnnotationAssertion(rdfs:seeAlso obo:RO_0016002 "https://github.com/oborel/obo-relations/issues/478")
 SubObjectPropertyOf(obo:RO_0016002 obo:RO_0016001)
 
+# Object Property: obo:RO_0018001 (is myristoyltransferase activity)
+
+AnnotationAssertion(obo:IAO_0000117 obo:RO_0018001 "Charles Tapley Hoyt"@en)
+AnnotationAssertion(dce:contributor obo:RO_0018001 <https://orcid.org/0000-0003-4423-4370>)
+AnnotationAssertion(rdfs:label obo:RO_0018001 "is myristoyltransferase activity"@en)
+SubObjectPropertyOf(obo:RO_0018001 obo:RO_0002564)
+
+# Object Property: obo:RO_0018002 (myristoylates)
+
+AnnotationAssertion(obo:IAO_0000116 obo:RO_0018002 "Axiomatization to GO to be added later"@en)
```

Comment:
Not sure how useful this is without a reference to a GitHub issue.

Reply:
Added reference to github issue in ae78197, but this note is the same as the one @cmungall left in the phosphorylates/ubiquitinates relations too

Reply:
Good to learn from the generals :D But we can do some things a bit better I believe.

Reply:
Ideally we would have dynamic import of GO driving this.  @matentzn  - how far are we from having this in place?